sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/CheckFileLicenseTest.cs

```diff
@@ -203,10 +204,18 @@ public void CheckFileLicense_WithForcingEmptyLines_ShouldBeCompliant_CS()
 
         [TestMethod]
         [TestCategory("Rule")]
-        public void CheckFileLicense_WhenLicensedWithMultiSingleLineCommentStartingWithNamespaceAndRegex_ShouldBeCompliant_CS()
+        public void CheckFileLicense_WhenLicensedWithMultiSingleLineCommentStartingWithNamespaceAndRegexMultiLine_ShouldBeCompliant_CS()
         {
             Verifier.VerifyAnalyzer(@"TestCases\CheckFileLicense_MultiSingleLineLicenseStartWithNamespace.cs",
-                new CheckFileLicense { HeaderFormat = MultiSingleLineRegexHeader, IsRegularExpression = true });
+                new CheckFileLicense { HeaderFormat = MultiLineRegexWithNewLine, IsRegularExpression = true });
+        }
+
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void CheckFileLicense_WhenLicensedWithMultiSingleLineCommentStartingWithNamespaceAndRegexSingleLine_ShouldBeCompliant_CS()
```

Comment:
[minor] as I mentioned before, you use `HeaderFormat = MultiLineRegexWithDot` and this test method name is `...RegexSingleLine_ShouldBeCompliant_CS`

Can you please align the test method name with the constant name? (e.g. `...RegexMultiLineWithDot_ShouldBeCompliant_CS`)

Reply:
Hope, this is as expected, now.

Reply:
thanks!